scripts/recolor.py

```diff
@@ -124,7 +124,7 @@ def recolor(flavor, accent: str):
     print("Mod SASS Color_Palette_default")
 
     # Greys
-    if flavor.name == "latte":  # Hardcode till someone smarter than me comes along
+    if flavor.name == PALETTE.latte.name:  # Hardcode till someone smarter than me comes along
```

Comment:
im assuming this comment is talking about what your changing
```suggestion
    if flavor.name == PALETTE.latte.name:
```